docs/meetings/2023/TechArea20230829.md

```diff
@@ -31,10 +31,23 @@ Triage duty shifts Tue-Mon
 OSG 23:
 
 -   AI (Matt): Update mash scripts to merge OSG and UW HTCondor repos (SOFTWARE-5653)
+    - Table in design doc for which htcss repos to merge with osg
+-   AI (Matt): Complete the two -contrib builds (reach out to Mat for what to do with repo) 
 -   AI (Matt): Kick off OSG 23 XRootD tests (worker node and XRootD)
+    - Most tests require the OSG-23 condor build, these two don't
```

Comment:
```suggestion
    - Most tests require the HTCSS-23 condor build, these two don't
```

We're not actually building HTCondor in the OSG 23 repos, we're depending on the dev team to release them and we'll be merging the two repos with mash (your task above)